### DISCUSSION TITLE
mailbyfilecoin:hope I can discuss a idea here.

### DISCUSSION THREAD

**Discussion by lincolnTeng** [filecoin-project/community#611] [General]
if it's not fit here , pls guide. thkx， 

as the name "mailbyfilecoin" talked.

it's came from several thought:
 
 1. blockchain is another Network， not another "Financial sector“ before it became a useful Network. email is a case runing in working NETWORK。
 2.  everybody use email ， even if  in web3 age.
 3.  email‘s work flow need network， but don't need permanent storage,  it COULD  start  & arrive  at endpoint client， while if it can be transfered properly.. and stored temparely...
 4. email could be realized in smartcontract based FILECOIN network， and I think it's the best fit network.
 5. email client could be combined in wallet ,and share the address as email address.

any discuss or help is welcomed,  

a typical work flow could be this.

1. client register to  a node of FILECOIN as the agent-server.
2. client send email to agent-server  as MTA , 
3. if a email sent to a client,  the node-MTA store it untile client registed on the network 
4. the MTA send email to the  dest-client's node-MTA
5. client connect node-MTA rective email & delete the email from node-MTA
6. take each process as a "DEAL" and pay it Automaticly...

hope I  made my idea clear ...

**Comment by Ipfs968fil** [comment]
可以啊！欢迎3.14来了！！！

**Comment by Ipfs968fil** [comment]
如何能让储存，更简单点。任何人一学就会用。
